Nodejs/Tests/AnalysisTests/AnalysisFile.cs

```diff
@@ -131,6 +131,20 @@ static class Analysis {
             List<AnalysisFile> files = new List<AnalysisFile>();
             foreach (var file in Directory.GetFiles(directory, "*", SearchOption.AllDirectories)) {
```

Comment:
I think this code should still be there for testing reason. 
Without that code we could measure just raw JsAnalyzer performance without any limitations, but there should be way to measure analysis speed with depth limit. If needed I could add command line switch to be able distinguish between desired behaviors.